README.md

```diff
@@ -305,6 +305,68 @@ value, if available. If there is no fallback value, the token will be omitted
 from the JavaScript objects. In general, if you want to share a value between
 CSS and JavaScript, you should use simple strings or numbers.
 
+### Width Based Media Queries
+
+Gesso uses custom mixins to specify viewport width based media queries:
+`breakpoint`, `breakpoint-max` and `breakpoint-min-max` for min-width,
```

Comment:
I think a bulleted list might be easier to read here:
* `breakpoint`: min-width queries
* `breakpoint-max`: max-width queries
* `breakpoint-min-max`: queries with both a min and max width